src/studio/src/designer/backend/Factories/ModelFactory/JsonMetadataParser.cs

```diff
@@ -108,6 +110,8 @@ private void CreateModelFromMetadataRecursive(Dictionary<string, string> classes
                     else
                     {
                         classBuilder.AppendLine("    [XmlElement(\"" + element.Value.XName + "\")]");
+                        classBuilder.AppendLine("    [JsonProperty(\"" + element.Value.XName + "\")]");
+                        classBuilder.AppendLine("    [JsonPropertyName(\"" + element.Value.XName + "\")]");
```

Comment:
Why do we need both?

Reply:
Ref discussion on slack, we use Newtonsoft.Json for deserialization, and System.Text.Json for serialization. We would like to replace Newtonsoft with System.Text eventually, but this is a big job and way outside the scope of this change.

Reply:
Added a comment in the code so we remember why we did this :)